### DISCUSSION TITLE
How to import osu!stable beatmaps into osu!lazer in the newest update?

### DISCUSSION THREAD

**Discussion by Jacie1krece** [ppy/osu#22538] [Q&A] [answered]
Hi, I know that the question sounds dumb, but the button for doing that disappeared from its position in the options menu. Was it moved, or it's missing entirely and I have to manually download all the songs I recently added to stable? Also, I tried to migrate osu!lazer to the same disk as osu!stable (stable's path is D:\osu! and lazer's path is D:\osu!lazer), and nothing changed.
![osu_2023-02-06_23-24-14](https://user-images.githubusercontent.com/33695007/217101292-fe7538af-5ec7-4dc2-87a3-818b556dd801.png)
![osu_2023-02-06_23-24-04](https://user-images.githubusercontent.com/33695007/217101283-8b7817ee-aa1c-4bf0-909d-9af24da10abc.png)

**Comment by Cootz** [comment, marked as answer]
If I remember correctly, it's dev-only feature now.
Idk if this is acceptable fix but you can run setup wizard and export all the maps from there:

![image](https://user-images.githubusercontent.com/50776304/217120710-4b42970a-1c2d-4ae8-840e-f9835703da4c.png)

**Reply by Jacie1krece** [reply]
It works. Thank you!
